### DISCUSSION TITLE
Clickable time distribution chart

### DISCUSSION THREAD

**Discussion by olympia** [algertc/ALPR-Database#20] [Ideas]
It would be great if clicking on the time distribution chart in the main dashboard could open those plates in a new window, especially when combined with the time selector feature on the dashboard.

**Comment by algertc** [comment]
I like this one a lot. I personally find the date selector very difficult to use, and it doesn't allow for hour-based selection either. Will def integrate this.

All the others very good too.

**Comment by olympia** [comment]
This became really nice, thank you!
